Add missing meta descriptions (#488)
mongodb/docs-golang: source/fundamentals/monitoring/cluster-monitoring.txt

.. meta::
   :keywords: code example, server, topology

>    :description: Monitor topology events in MongoDB using the Go driver to track cluster changes, assess health, and perform capacity planning by subscribing to SDAM events.

Reviewer: ```suggestion
   :description: Monitor topology events with the MongoDB Go Driver to track cluster changes, assess health, and perform capacity planning by subscribing to SDAM events.
```
